Automattic/wp-calypso
# ci(e2e): Move Gutenberg e2e tests to its own suite

#### Changes proposed in this Pull Request

* Move Gutenberg e2e tests to `specs-gutenberg/`
* Change suite `magellan.json` to include `specs-gutenberg/`. This is the suite run by CircleCI, nothing should change there.
* Create a new suite `magellan-calypso.json` without the Gutenberg tests. This suite will be run by TeamCity.

#### Testing instructions

* Verify CircleCI e2e tests are green and run the same number of tests than `trunk`.
* Verify TeamCity e2e tests are green and run less tests

Edit: Note TeamCity e2e test are failing because this build runs less than 50% than the previous green build (which is expected). Otherwise the tests are passing. I've marked the build as 'passing' to avoid confusion.
